### ISSUE TITLE
Define the appropriate loss function for the model using the validation data.

### ISSUE THREAD

**Issue by isaaclucky** [Ad-Campaign-Performance/abtest-mlops#7]
Compute feature importance 
- what’s driving the model? 
- Which parameters are important predictors for the different ML models? 
- What contributes to the goal of gaining more “Yes” results?
- Which data features are relevant to predicting the target variable?
- For each of the ML algorithms above, find the best model by tuning their hyperparameters and each time adding the tried models in MLFlow.
